### PR TITLE
Logging improvements for volume requests

### PR DESCRIPTION
**Pull request by MasslessParticle**
I noticed we were logging volume requests as metric requests in `metrics.go`. This PR logs volume requests on their own with relevant data.

### CODE REVIEW

File: pkg/logql/metrics.go [hunk 1]
```diff
@@ -288,6 +289,52 @@ func RecordSeriesQueryMetrics(
...
+		"status", status,
+		"splits", stats.Summary.Splits,
+		"total_entries", stats.Summary.TotalEntriesReturned,
+	)
+
+	sharded := strconv.FormatBool(false)
+	if stats.Summary.Shards > 1 {
+		sharded = strconv.FormatBool(true)
+	}
+	bytesPerSecond.WithLabelValues(status, queryType, "", latencyType, sharded).
```
Comment (trevorwhitney) [inline pkg/logql/metrics.go:326]: can we add `		"throughput", strings.Replace(humanize.Bytes(uint64(stats.Summary.BytesProcessedPerSecond)), " ", "", 1),` to the log line as well?

Comment (trevorwhitney) [reply pkg/logql/metrics.go:326]: and how about `total_bytes` too? I haven't dug in to if we're tacking these correctly, but if not we should remove the metric, and if so we should add the log line.

**Review by trevorwhitney — APPROVED** [review]
can we add `throughput` too?

**Comment by MasslessParticle** [comment]
I removed irrelevant metrics, instead. Those are pretty specific to queries of loglines

**Pull request merged**